tests/_core/search/rest_total_hits_as_int.yaml

```diff
@@ -0,0 +1,125 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test search endpoint with rest_total_hits_as_int
```

Comment:
Add a period. I started adding a linter in https://github.com/opensearch-project/opensearch-api-specification/pull/463.